php/Dockerfile

```diff
@@ -69,4 +69,7 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
 # Install Grunt CLI
```

Comment:
We could also upgrade nodejs to v10.x at line 66 ?
```RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \```

Reply:
@maxhelias Yes, i just change that